### DISCUSSION TITLE
Discussions!

### DISCUSSION THREAD

**Discussion by lue-bird** [lue-bird/elm-typesafe-array#1] [General]
## 👋 Welcome!
  * Ask questions you’re wondering about.
  * Share ideas.
  * be open-minded.

**Comment by sammypuyoy** [comment]
so i use the phex zip since the new extensions just does not work and the phex zip extension is not working is there a fix?

**Comment by sammypuyoy** [comment]
Can you please bring back the old extension it was the only one that worked for me and a few of my other friends can't use it either it would be super nice and kind if you brought the other one back. Thank you :)

**Reply by lue-bird** [reply]
It seems like you posted a comment on the wrong discussion. This elm package has nothing to do with extensions.
